src/qibolab/platform.py

```diff
@@ -137,14 +142,19 @@ def _execute(self, method, sequences, options, **kwargs):
         log.info(f"Minimal execution time (seq): {time}")
 
         result = {}
+
         for instrument in self.instruments.values():
             if isinstance(instrument, Controller):
-                new_result = getattr(instrument, method)(self.qubits, sequences, options)
+                if self.couplers is not None:
+                    new_result = getattr(instrument, method)(self.qubits, self.couplers, sequences, options)
+                else:
+                    new_result = getattr(instrument, method)(self.qubits, sequences, options)
```

Comment:
Couldn't you always pass `self.couplers`, just ignore if they are `None`?

E.g. just use:
```suggestion
                new_result = getattr(instrument, method)(self.qubits, sequences, options, couplers=self.couplers)
```

Using `**kwargs` in the functions to thrash extra arguments.

Reply:
I also made a comment about that above and we ended up not changing it because it would require to change the `play` and `sweep` methods in all the drivers. As you say, it only requires adding a `**kwargs`, so it isn't that big of a change. The idea was that at some point we would add support for couplers in more drivers and this would be fixed. But this may take time (if it ever happens), so it may be worth having a fixed interface in the meantime.

Reply:
Should I leave like this then ?

Reply:
Yes, then let's do as @stavros11 said. But open an issue even about this one, because it is something we are aware we should do.

Reply:
Here https://github.com/qiboteam/qibolab/issues/610.